webpack/scenes/ContentViews/Table/emptyRows.js

```diff
@@ -0,0 +1,35 @@
+// Meant for when the user has none of the object created, not when a search returns empty results
+import React from 'react';
+import { EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Bullseye,
+  Title } from '@patternfly/react-core';
+import { CubeIcon } from '@patternfly/react-icons';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const emptyRows = [{
```

Comment:
This was mostly taken from the patternfly [documentation](https://patternfly-react.surge.sh/documentation/react/components/table#empty-state)

With the empty state as a row, the table headers still show, which is also what the current CV page does when you have no CVs.

The current approach seems like the best way to show the table headers, where having it in a new component would not unless a new table was created. I prefer the table headers being there even with empty state as it give context to the page. 

@rohoover are there any design guidelines around this? (Showing table headers when there  is no content in the table, only the empty rows message)

Reply:
@johnpmitsch Even if we keep the current structure with the table headers, I would still prefer the `title` JSX to be its own component and file.  Seems cleaner to me to separate JSX from table infrastructure data wherever possible.  So instead it could be, for example
```js
title: <TableEmptyState />
```

Reply:
At least I'm not making you use `React.createElement` :smile:

Reply:
Hey guys - guidance for empty states is here:
https://docs.google.com/document/d/1K6q0rQpOc9qOetHp1Zx7rHpnbKpPpDCfGIjhv6pja0U/edit#heading=h.j9odg8ohi1sx

To answer your question specifically, if the empty state exists because there are no Content Views created then we should have a full page empty state with no table headers -- call to action button for "Create content view". If we have an empty state because we filtered out all possible results, then it should be included in the table with headers.

Reply:
Thanks @rohoover! Here is what I changed it to. The "Create New Content View" button will be added in a later PR, so its just the message for now.

![here](http://www.clipular.com/c/5305342542741504.png?k=w84TmuVL8udn2A33nfd_ku6yzaY)

@jeremylenz Let me know if the recent update is what you had in mind for what the component renders.